core/modules/locale/locale.pages.inc

```diff
@@ -124,18 +124,23 @@ function _locale_translate_seek() {
 /**
  * List languages in search result table
  */
-function _locale_translate_language_list($translation, $limit_language) {
+function _locale_translate_language_list($string, $limit_language) {
```

Comment:
I can't tell why the `$translation` parameter here was changed to `$string`. Previously, `$string['languages']` was passed in as the value of `$translation`. In the new code, `$string` is passed in entirely, but only `$string['languages']` is used in the remainder of the function. Could this parameter be left as-is to reduce the changed code?